internal/provider/resource_tf_importer.go

```
@@ -317,7 +319,9 @@ func writeTfState(ctx context.Context, resources []instanceData, outputPath stri
 	tflog.Info(ctx, fmt.Sprintf("Running %q in %s", stateReplaceCommandStr, outputPath))
 
 	if err = cmd.Run(); err != nil {
-		return diag.Errorf("failed to run terraform CLI command: %q. Please run it manually in %s.", stateReplaceCommandStr, outputPath)
+		return diag.Errorf("The process was successfully completed, but there's one more manual step you'll "+
```

ditto

---

same thing here, if the command failed to run, we ask a user to run it manually.

> cmd.Run() may might fail for a few reasons, such as the command not being found on the system, lacking the proper permissions to execute it, or the command itself exiting with an error state. For example, if you try to execute a non-existent command like cmd := exec.Command("someNonExistentCommand"), cmd.Run() will return an error because the system can't find or execute the specified command. Another example is trying to execute a script that exits with a status code indicating an error, which cmd.Run() will also catch and return as an error.

By default, Terraform hides logs from the user, so the only way to provide a helpful, actionable message is through error output.